DashboardFrontend/Controller.cs

```diff
@@ -354,56 +350,141 @@ private void GetCredentials()
         }
 
         /// <summary>
-        /// Updates the list of executions in the current conversion.
+        /// Gathers initial information (Executions, health report info) and then starts the query timers that determine when to fetch data for the individual parts of the Conversion.
+        /// The main loop ensures that data is processed when appropriate, as determined by the timers.
         /// </summary>
-        private void UpdateExecutions()
+        /// <remarks><see cref="Thread.Sleep(int)"/> is used to drastically reduce the number of times the loop is executed, lowering the overall CPU load.</remarks>
+        private void StartMonitoring()
         {
-            if (IsUpdatingExecutions || Conversion is null)
+            _vm.IsRunning = true;
+            UpdateExecutions();
+            DU.BuildHealthReport(Conversion?.HealthReport);
+            SetupTimers();
+
+            while (_vm.IsRunning)
             {
-                return;
+                if (ShouldUpdateLog)
+                {
+                    Trace.WriteLine("log");
+                    UpdateLog();
+                    ShouldUpdateLog = false;
+                }
+                if (ShouldUpdateManagers)
+                {
+                    Trace.WriteLine("mgr");
+                    UpdateManagerOverview();
+                    ShouldUpdateManagers = false;
+                }
+                if (ShouldUpdateValidations)
+                {
+                    Trace.WriteLine("vr");
+                    UpdateValidationReport();
+                    ShouldUpdateValidations = false;
+                }
+                if (ShouldUpdateHealthReport)
+                {
+                    Trace.WriteLine("hr");
+                    UpdateHealthReport();
+                    ShouldUpdateHealthReport = false;
+                }
+                if (ShouldUpdateExecutions)
+                {
+                    Trace.WriteLine("e");
+                    UpdateExecutions();
+                    ShouldUpdateExecutions = false;
+                }
+                Thread.Sleep(10);

```

Comment:
er det nødvendigt med 10? 
```suggestion
                Thread.Sleep(100);

```